travis/ubuntu-compile-nsenter.sh

```diff
@@ -17,7 +17,9 @@ set -e
 
 sudo apt-get update
 sudo apt-get install libncurses5-dev libslang2-dev gettext zlib1g-dev libselinux1-dev debhelper lsb-release pkg-config po-debconf autoconf automake autopoint libtool
-mkdir .tmp || true
-wget https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.2.tar.gz -qO - | tar -xz -C .tmp/
-cd .tmp/util-linux-2.30.2 && ./autogen.sh && ./configure && make nsenter
+mkdir $HOME/nsenter || true
```

Comment:
You can use `-p` instead of `|| true`:

```
$ mkdir $HOME
mkdir: cannot create directory ‘/home/jpkroehling’: File exists
$ mkdir -p $HOME
$ echo $?
0
```